drivers/performance_measurement/app/models/performance_measurement/report.rb

```diff
@@ -541,9 +546,8 @@ def can_see_client_details?(user)
         'Measure 5',
         'Measure 7',
       ]
-      # For now, we're using a fixed set of project types
+
       options = filter.to_h
-      options[:project_type_codes] = [:es, :so, :sh, :th, :ph]
```

Comment:
Are these actually correct for the SPM @eanders? I remember you mentioned you didn't think one of them should be there.

Reply:
Yes, you were correct, SO _is_ included.